packages/jellyfish-api-core/__tests__/category/governance/listGovProposalVotes.test.ts

```diff
@@ -67,6 +67,52 @@ describe('Governance', () => {
     expect(votes[0].vote).toStrictEqual('YES')
   })
 
+  it('should paginate', async () => {
+    const data = {
+      title: 'A vote of confidence',
+      context: '<Git issue url>'
+    }
+    const proposalId = await testing.rpc.governance.createGovVoc(data) // Creates a vote of confidence on which to vote
+    await testing.container.generate(1)
+
+    for (const [id, data] of Object.entries(masternodes)) {
+      if (data.operatorIsMine) {
+        await testing.container.generate(1, data.operatorAuthAddress) // Generate a block to operatorAuthAddress to be allowed to vote on proposal
+        await testing.rpc.governance.voteGov({
+          proposalId,
+          masternodeId: id,
+          decision: VoteDecision.YES
+        })
+      }
+    }
+    await testing.container.generate(1)
+
+    const votes = await testing.rpc.governance.listGovProposalVotes({ proposalId: proposalId })
+    const votes1 = await testing.rpc.governance.listGovProposalVotes({
+      proposalId: proposalId,
+      pagination: {
+        start: 0,
+        including_start: true,
+        limit: 2
+      }
+    })
+    const votes2 = await testing.rpc.governance.listGovProposalVotes({
+      proposalId: proposalId,
+      pagination: {
+        start: 2,
+        including_start: true,
+        limit: 2
+      }
+    })
```

Comment:
apart from this test, would be also good to test the different combinations of pagination options (eg. `start` is omitted, `including_start` is omitted or false, `limit` is omitted or 0)

Reply:
Fixed: https://github.com/JellyfishSDK/jellyfish/pull/1908/commits/09c483e9cd769cbf47fbce486d20bca448c6bf91

Reply:
The test should be in separated `it()`s describing the pagination condition.